documentation/docs/getting_started/hornet_apt_repository.md

```diff
@@ -104,7 +104,6 @@ All installation methods mentioned in this article from this point should be con
    Please make sure to download the binaries for your system:
    
    * `HORNET-X.Y.Z_Linux_x86_64.tar.gz`: standard 64-bit-linux-based executable, such as Ubuntu, Debian, etc.
-   * `HORNET-X.Y.Z_Linux_arm64.tar.gz`: executable for Raspberry Pi 4.
```

Comment:
I wouldn't remove it, but rewrite it to `executable for 64bit ARM based systems.`